src/bitpacking.rs

```diff
@@ -236,6 +262,37 @@ impl_packing!(u16);
 impl_packing!(u32);
 impl_packing!(u64);
 
+macro_rules! impl_packing_compare {
+    ($T:ty) => {
+        paste! {
+            impl BitPackingCompare for $T {
+               #[inline(always)]
+                fn unpack_cmp_impl<const W: usize, F: Fn(Self, Self) -> bool>(
+                    input: &[Self; 1024 * W / Self::T],
+                    output: &mut [Self; 1024 / Self::T],
+                    f: F,
+                    other: Self,
+                ) where BitPackWidth<W>: SupportedBitPackWidth<Self> {
+                    for lane in (0..Self::LANES){
+                        unpack!($T, W, input, lane, |$idx, $elem| {
+                            let bool_idx = $idx / Self::T;
```

Comment:
Can you add a dump of the ASM to the PR for e.g. `3-bit => 32-bit GTE`

Reply:
Included